File: android/OhDormitory/app/src/main/java/kr/hs/emirim/uuuuri/ohdormitory/Model/Score.java
package kr.hs.emirim.uuuuri.ohdormitory.Model;

/**
 * Created by 유리 on 2017-10-04.
 */

public class Score {
    private String detail;
    private Double score;

    public String getDetail() {
        return detail;
    }

    public Double getScore() {
        return score;
    }

    public Score(String detail, Double score) {
        this.detail = detail;
        this.score = score;
    }
}

File: ios/OhDormitory/OhDormitory/Score.swift
//
//  Score.swift
//  OhDormitory
//
//  Created by 장명수 on 2018. 6. 10..
//  Copyright © 2018년 doori. All rights reserved.
//

import Foundation

class Score {
    
    
    var score_id:Int
    var detail:String
    var score:Double
    init(score_id:Int, detail:String, score:Double){
        self.score_id = score_id
        self.detail = detail
        self.score = score
    }
}

